telemetry/definitions/commonDefinitions.json

```diff
@@ -177,6 +178,11 @@
             "type": "string",
             "allowedValues": ["IAM", "SecretsManager"],
             "description": "How the database credentials are being retrieved"
+        },
+        {
+            "name": "sqsQueueType",
+            "allowedValues": ["standard", "fifo"],
+            "description": "The type of an SQS Queue (either standard or fifo)"
```

Comment:
nit: if we have valid values enumerated programmatically, better to leave them out of descriptions (or auto-generate the description). Small thing, but adds up to a maintenance cost over time

Reply:
Sure, removed from the description